tools/generate-helpers/pg-table-bindings/store.go.tpl

```diff
@@ -205,6 +205,8 @@ func {{ template "insertFunctionName" $schema }}({{ if eq (len $schema.Children)
         pgutils.NilOrTime({{$field.Getter "obj"}}),
         {{- else if eq $field.SQLType "uuid" }}
         pgutils.NilOrUUID({{$field.Getter "obj"}}),
+        {{- else if eq $field.DataType "map" }}
+        pgutils.EmptyOrMap({{$field.Getter "obj"}}),
```

Comment:
Since there was a change in how maps are serialized (`null` vs `'null'`) I've decided to save them as empty object `{}` no matter if they are null or empty.

Reply:
Extracted 
- https://github.com/stackrox/stackrox/pull/6929